tests/test_preprocess.py

```diff
@@ -104,6 +104,14 @@ def test_assign_index(index):
     assert pc.dataset.index[0] != 0
 
 
+def test_duplicate_columns():
```

Comment:
Is this test related to the change to the encoder? Do we need to add a test for the encoding issue?

Reply:
it's unrelated. just a minor check I added. I don;t think we need a test for the encoding since it's no longer an issue. Only if users would select specifically the LeaveOneOut encoder, which will only happen if they understand the encoder, in which case they should see it doesn;t work as expected